src/aleph_client/commands/program.py

```diff
@@ -83,8 +84,36 @@ def upload(
         help="Hash of the runtime to use for your program. Defaults to aleph debian with Python3.8 and node. You can also create your own runtime and pin it",
     ),
     beta: bool = typer.Option(False),
+
+
+
     debug: bool = False,
     persistent: bool = False,
+    persistent_volumes: Optional[Tuple[str, str, str]] = typer.Option(
```

Comment:
With this it is currently not possible to have multiple persistent volumes. I'd also recommend renaming it to singular, as what the user sees on the CLI would be more like:
`aleph program --persistent_volume host my-volume 1 --persistent_volume host another-volume 1`

```suggestion
    persistent_volume: Optional[List[Tuple[str, str, str]]] = typer.Option(
```